src/plugins/telemetryTable/components/table.vue

```diff
@@ -631,8 +691,110 @@ export default {
         clearRowsAndRerender() {
             this.visibleRows = [];
             this.$nextTick().then(this.updateVisibleRows);
-        }
+        },
+        pause(pausedByButton) {
```

Comment:
Why do we care whether it was paused by the button, or by marking a row? The logic looks the same either way to me?